lib/foreman_rh_cloud/engine.rb

```diff
@@ -128,6 +128,14 @@ class Engine < ::Rails::Engine
         description: N_('Run remediation playbook generated by Insights'),
         host_action_button: false
       )
+
+      unless ForemanTasks::RecurringLogic.joins(:tasks).merge(
+        ForemanTasks::Task.where(label: 'InventorySync::Async::InventoryScheduledSync')
+      ).exists?
+        recurring_logic = ForemanTasks::RecurringLogic.new_from_cronline("0 0 * * *")
+        recurring_logic.save!
+        recurring_logic.start(InventorySync::Async::InventoryScheduledSync)
```

Comment:
`#start` calls `#start_after`, which calls `#save!` under the hood so you should be able to reduce this to
```suggestion
        ForemanTasks::RecurringLogic.new_from_cronline("0 0 * * *")
                                    .start(InventorySync::Async::InventoryScheduledSync)
```

Reply:
For some reason it didn't save the logic unless I saved it manually. Should I debug it further, or leaving it as it is right now is also OK?